server.js

```diff
@@ -14,35 +15,28 @@ app.get('/info', (req, res) => {
 })
 
 app.get('/products/all', (req, res) => {
-    res.json(products)
+    let productsMap = dataService.getProducts()
+    let productsArray = []
+
+    let getValues = (value, key, productsMap) => {productsArray.push(value)}
+
+    productsMap.forEach(getValues)
+
+    res.json(productsArray)
```

Comment:
Should return a map of products, not just an array of product objects. See above json example.

Reply:
Can I restart the project? In a fresh repository...

Reply:
How do I send a Map as a JSON response?
![image](https://user-images.githubusercontent.com/46944485/71885083-e69a7300-3139-11ea-88af-f009ac580763.png)
That is all my browser is showing when I send my Map as a JSON response.

Reply:
It actually just send a map object as you suggested but it only `res.json()` takes only JS objects and array. All attempts to convert the map to objects have proved futile.